docs/remote-runners.md

```diff
@@ -26,6 +26,4 @@ When using remote runners, even though the build executes remotely, the followin
 
 To get started with free remote runners managed by Earthly, check out [Earthly Satellites](cloud/satellites.md).
 
-To get started with self-managed remote runners, see
-* The [remote buildkit page](ci-integration/remote-buildkit.md)
-* The [Kubernetes setup page](ci-integration/guides/kubernetes.md)
+To get started with self-hosted runners, see the [Self-Hosted Satellites Guide](cloud/satellites/self-hosted.md).
```

Comment:
I wouldn't remove our open-source offering here. Perhaps this is where we have a little comparison between self-hosted satellites vs remote Buildkit. See a comparison example here: https://docs.earthly.dev/docs/caching#sharing-cache. Perhaps we can reuse the format / style of it.

Reply:
I added the paragraph below to set context and link to the remote Buildkit page. It mentions the comparison  table you recommended. So that needs to be added if we want to keep the below paragraph.